### PR TITLE
DEV-3515-feat(piscine-go): add README.md for new go exercise stringtointarray

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added the README.md for the new go exercise stringtointarray.

### CODE REVIEW

**Review by jotapero — APPROVED** [review]

File: subjects/stringtointarray/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## stringtointarray
```
Comment (eslopfer) [inline subjects/stringtointarray/README.md:1]: It's not an array, it's a slice. The name is wrong.

File: subjects/stringtointarray/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Instructions
+
+Write a function `StringToIntArray()` that takes a `string` and returns the corresponding `int` array.
```
Comment (eslopfer) [inline subjects/stringtointarray/README.md:5]: Same as above

File: subjects/stringtointarray/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+func main() {
+  fmt.Println(piscine.StringToIntArray("A quick brown fox jumps over the lazy dog"))
+  fmt.Println(piscine.StringToIntArray("Converted this string into an int"))
+  fmt.Println(piscine.StringToIntArray("hello THERE"))
+}
+```
+
+And its output:
+
+```go
```
Comment (eslopfer) [inline subjects/stringtointarray/README.md:36]: `console` instead of `go`

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/stringtointarray/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/stringtointarray/README.md]: It is fixed.

File: subjects/stringtointarray/README.md (hunk 2 above)
Comment (tiagocollot) [reply subjects/stringtointarray/README.md]: It is fixed.

File: subjects/stringtointarray/README.md (hunk 3 above)
Comment (tiagocollot) [reply subjects/stringtointarray/README.md]: It is fixed.

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**